hr_expense_sequence/__init__.py

```diff
@@ -0,0 +1,12 @@
+from . import models
```

Comment:
Copyright/License would be great

Reply:
I think it is not necessary on init files.

Reply:
Well, copyright - no, license - yes. But this file also contains code - that should've been separate and with copyright.

Reply:
Well this a small hole :cry:
I have seen PR asking for the removal. I think the best would be to have a global agreement on OCA. However, It is probably enough with the license

Reply:
Yeah, it's just matter of keeping things clean and making them cleaner